.workflow-version

```diff
@@ -0,0 +1 @@
+$workflow_version='1.0.0.alpha1'
```

Comment:
Should we follow [Chromium's approach](https://github.com/chromium/chromium/blob/f2247f1b3040c9831f950d1c5523ea20b3157a1a/chrome/VERSION) here and break the version into its constituent parts? That makes it _really_ easy to e.g. update from automated tooling.

Reply:
Good idea, it'll help make our automation cleaner as we keep going.